src/lib/y2storage/boot_requirements_strategies/base.rb

```diff
@@ -84,7 +85,8 @@ def needed_partitions(target)
       def warnings
         res = []
 
-        if !boot_readable_by_grub?
+        if !boot_readable_by_grub? &&
+            !Y2Storage::BootRequirementsStrategies::BLS.bls_bootloader_proposed?
```

Comment:
It is strange to have a base class with knowledge about its derivated classes. I would add a new method in the base class (e.g., `#grub_warning?`), which can be redefined in `BLS` class to always return false.